docs/source/class_basics.rst

```diff
@@ -233,6 +233,12 @@ show an error:
        def g(self, y: str) -> None:   # Error: no corresponding base method found
            ...
 
+.. note::
+
+   Use ``--strict-override-decorator`` or
+   :confval:`strict_override_decorator = True <strict_override_decorator>` to require
+   methods overrides use the ``@override`` decorator. Emit an error if it is missing.
```

Comment:
"methods overrides use" -> "that method overrides use"?

Mention above that the decorator is included in 3.12 and must be imported from `typing_extensions` if using an earlier Python version, in particular since 3.12 is not out yet.